eth2/utils/ssz/README.md

```diff
@@ -1,543 +1,3 @@
-
-Deserializes the "length" value in the serialized bytes from the index. The
-length of bytes is given (usually 4 stated in the reference implementation) and
-is often the value appended to the list in front of the actual serialized
-object.
-
-|     Parameter    | Description                                |
-|:----------------:|:-------------------------------------------|
-|     ``bytes``    | Serialized list of bytes.                  |
-|     ``index``    | Starting index to deserialize from.        |
-| ``length_bytes`` | Number of bytes to deserialize into usize. |
-
-
-**Returns**
-
-|   Return Value  | Description                                                |
-|:---------------:|:-----------------------------------------------------------|
-|    ``usize``    | The length of the serialized object following this length. |
-| ``DecodeError`` | Error if the decoding could not be performed.              |
-
-**Example**
-
-```rust
-let length_of_serialized: Result<usize, DecodeError> = decode_length(&encoded, 0, 4);
-```
-
----
-
-## Usage
-
-### Serializing/Encoding
-
-#### Rust
-
-Create the `simpleserialize` stream that will produce the serialized objects.
-
-```rust
-let mut ssz = SszStream::new();
-```
-
-Encode the values that you need by using the ``append(..)`` method on the `SszStream`.
-
-The **append** function is how the value gets serialized.
-
-```rust
-let x: u64 = 1 << 32;
-ssz.append(&x);
-```
-
-To get the serialized byte vector use ``drain()`` on the `SszStream`.
-
-```rust
-ssz.drain()
-```
-
-**Example**
-
-```rust
-// 1 << 32 = 4294967296;
-// As bytes it should equal: [0,0,0,1,0,0,0]
-let x: u64 = 1 << 32;
-
-// Create the new ssz stream
-let mut ssz = SszStream::new();
-
-// Serialize x
-ssz.append(&x);
-
-// Check that it is correct.
-assert_eq!(ssz.drain(), vec![0,0,0,1,0,0,0]);
-```
-
-## Deserializing/Decoding
-
-#### Rust
-
-From the `simpleserialize` bytes, we are converting to the object.
-
-```rust
-let ssz = vec![0, 0, 8, 255, 255, 255, 255, 255, 255, 255, 255];
-
-// Returns the result and the next index to decode.
-let (result, index): (u64, usize) = decode_ssz(&ssz, 3).unwrap();
-
-// Check for correctness
-// 2**64-1 = 18446744073709551615
-assert_eq!(result, 18446744073709551615);
-// Index = 3 (initial index) + 8 (8 byte int) = 11
-assert_eq!(index, 11);
-```
-
-Decoding a list of items:
-
-```rust
-// Encoded/Serialized list with junk numbers at the front
-let serialized_list = vec![ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 0, 0, 0, 32, 0, 0, 0,
-                            0, 0, 0, 0, 15, 0, 0, 0, 0, 0, 0, 0, 15, 0, 0, 0, 0,
-                            0, 0, 0, 15, 0, 0, 0, 0, 0, 0, 0, 15];
-
-// Returns the result (Vector of usize) and the index of the next
-let decoded: (Vec<usize>, usize) = decode_ssz_list(&serialized_list, 10).unwrap();
-
-// Check for correctness
-assert_eq!(decoded.0, vec![15,15,15,15]);
-
-assert_eq!(decoded.1, 46);
-```
+eth2_ssz = "0.1.2"
```

Comment:
I was thinking a crates.io "badge", like this one:

![Crates.io](https://img.shields.io/crates/v/eth2_ssz)

```
![Crates.io](https://img.shields.io/crates/v/eth2_ssz)
```

FYI, I made it at https://shields.io/ :)

Reply:
No worries at all, I wasn't very clear :)